notebooks/lesson09-overfitting.ipynb

```diff
@@ -0,0 +1,2113 @@
+{
```

Comment:
<p>typo: <code>cross_validate</code>  instead of "Cross-valid"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>